build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/test/rest/RestTestBasePlugin.java

```diff
@@ -114,8 +114,12 @@ public void apply(Project project) {
             task.dependsOn(integTestDistro, modulesConfiguration);
             registerDistributionInputs(task, integTestDistro);
 
+            // Enable parallel execution for these tests since each test gets its own cluster
             task.setMaxParallelForks(task.getProject().getGradle().getStartParameter().getMaxWorkerCount() / 2);
 
+            // Disable test failure reporting since this stuff is now captured in build scans
```

Comment:
I don't see the output in the build scan (or locally). I forced a failure and ran a test and by default don't see the information in the build scan, but can hard code it to true and see the expected output. 

default (as here) : https://gradle-enterprise.elastic.co/s/5i2ygpfhqv7hs/console-log?page=1#L326
hardcoded to true: https://gradle-enterprise.elastic.co/s/bzlst4wgyl7ys/console-log?page=1#L362

reproduce: 
```diff
diff --git a/build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/test/rest/RestTestBasePlugin.java b/build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/test/rest/RestTestBasePlugin.java
index c42f7a533f6..9316cb80a55 100644
--- a/build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/test/rest/RestTestBasePlugin.java
+++ b/build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/test/rest/RestTestBasePlugin.java
@@ -118,7 +118,7 @@ public class RestTestBasePlugin implements Plugin<Project> {
             task.setMaxParallelForks(task.getProject().getGradle().getStartParameter().getMaxWorkerCount() / 2);
 
             // Disable test failure reporting since this stuff is now captured in build scans
-            task.getExtensions().getExtraProperties().set("dumpOutputOnFailure", false);
+            task.getExtensions().getExtraProperties().set("dumpOutputOnFailure", true);
 
             // Disable the security manager and syscall filter since the test framework needs to fork processes
             task.systemProperty("tests.security.manager", "false");
diff --git a/x-pack/qa/core-rest-tests-with-security/src/yamlRestTest/java/org/elasticsearch/xpack/security/CoreWithSecurityClientYamlTestSuiteIT.java b/x-pack/qa/core-rest-tests-with-security/src/yamlRestTest/java/org/elasticsearch/xpack/security/CoreWithSecurityClientYamlTestSuiteIT.java
index 81533552765..1f06c16bf11 100644
--- a/x-pack/qa/core-rest-tests-with-security/src/yamlRestTest/java/org/elasticsearch/xpack/security/CoreWithSecurityClientYamlTestSuiteIT.java
+++ b/x-pack/qa/core-rest-tests-with-security/src/yamlRestTest/java/org/elasticsearch/xpack/security/CoreWithSecurityClientYamlTestSuiteIT.java
@@ -55,7 +55,7 @@ public class CoreWithSecurityClientYamlTestSuiteIT extends ESClientYamlSuiteTest
 
     @Override
     protected Settings restClientSettings() {
-        String token = basicAuthHeaderValue(USER, new SecureString(PASS.toCharArray()));
+        String token = basicAuthHeaderValue(USER+"x", new SecureString(PASS.toCharArray()));
         return Settings.builder().put(super.restClientSettings()).put(ThreadContext.PREFIX + ".Authorization", token).build();
     }
```
with 
```
./gradlew ':x-pack:qa:core-rest-tests-with-security:yamlRestTest' --tests "org.elasticsearch.xpack.security.CoreWithSecurityClientYamlTestSuiteIT" -Dtests.method="test {yaml=cluster.desired_nodes/10_basic/Test processors range min rejects 0.0}" -Dtests.seed=41ACFA3982E5C070 -Dtests.locale=sv-SE -Dtests.timezone=US/Pacific
```

Reply:
The cluster output is now longer in the console log, it's under the failed test itself: https://gradle-enterprise.elastic.co/s/5i2ygpfhqv7hs/tests/:x-pack:qa:core-rest-tests-with-security:yamlRestTest/org.elasticsearch.xpack.security.CoreWithSecurityClientYamlTestSuiteIT?top-execution=1

Reply:
Ahh... I see it now thanks. I even looked for it and could not find it... if anyone else comes across this here is where you click 

<img width="898" alt="image" src="https://user-images.githubusercontent.com/976291/210614407-2cad227e-1fcf-4d67-9f8e-5e0e5822c623.png">